Gemfile

```diff
@@ -74,6 +75,38 @@ else
       gem 'msgpack', '1.7.2'
     end
   end
+else
+
+  puts "FINAL_PACKAGE"
```

Comment:
@jmarrec can you explain the intent of "FINAL_PACKAGE"? It seems like this effectively means OpenStudio 3.7.0 series. aka pre ruby3 update.

Reply:
@jmarrec on the not "final package" branch of this logic it seems to me that we will need to update all of the `:ref  => ruby3` lines of code to develop after all of the merges happen. yes?